core/models.py

```diff
@@ -17,6 +17,7 @@
 
 class McUser(models.Model):
   """Fields from default User model:
+    title
```

Comment:
Not needed - these fields were for the fields we were going to use from the base User model. If you could get rid of 'first_name' and 'last_name' that would be great, I forgot to remove them lol.